### PR TITLE
feat: add representer version 1

### PR DESCRIPTION
**Pull request by vaeng**
See README.md

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 FROM alpine:3.17
```
Comment (ErikSchierboom) [inline Dockerfile]:
```suggestion
FROM alpine:3.18
```

File: tests/example-empty-file/representation.json
Comment (ErikSchierboom) [inline tests/example-empty-file/representation.json:1]: Should this file be named `expected_representation.json` (same goes for the other tests)?

**Review by ErikSchierboom — COMMENTED** [review]
Great!

Side note: we've dropped the `example-` prefix for the sub-directories in the `tests` dir, you could consider doing the same.

File: tests/example-empty-file/representation.json
Comment (vaeng) [reply tests/example-empty-file/representation.json:1]: Yes, I wondered about that at some point but never investigated further. I took it from the template. It is renamed now and the code for testing is up as well. I added a line in the .gitignore to skip generated files. Do you know if this overlook came from the template?

Comment (ErikSchierboom) [reply tests/example-empty-file/representation.json:1]: Yes, this is a bug! I'll fix

**Review by ErikSchierboom — APPROVED** [review]
We need to run some scripts to get this running on our infrastructure. I'll let you know when that is done, after which you can enable the representer in the track's `config.json`

**Comment by vaeng** [comment]
> We need to run some scripts to get this running on our infrastructure. I'll let you know when that is done, after which you can enable the representer in the track's `config.json`

Okay. Thanks. Should we merge this in the meantime?

**Comment by ErikSchierboom** [comment]
Yeah merging is fine!

**Comment by iHiD** [comment]
> We need to run some scripts to get this running on our infrastructure. I'll let you know when that is done, after which you can enable the representer in the track's config.json

@ErikSchierboom This will also be the first time a representer replaces the generic one. We'll need to back queue all C++ published solutions to rerun them on this. Let's discuss that on Slack when you're ready. @vaeng Please don't toggle that flag until we've worked this one out. Thanks! :)

**Pull request merged**